domains/corruptedarc.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Ishaan1313",
+           "email": "89083494+Ishaan1313@users.noreply.github.com",
+           "discord": "692993163333337139"
+        },
+    
+        "record": {
+            "CNAME": "phones-sells.gl.at.ply.gg:50105"
```

Comment:
You must remove the :50105 here, CNAME records cannot contain that.

Reply:
Its actually a CasaOS server